packages/wallets/.eslintrc.js

```diff
@@ -31,5 +35,13 @@ module.exports = {
                 ],
             },
         ],
+        'simple-import-sort/exports': 'error',
+        'import/first': 'error',
+        'import/newline-after-import': 'error',
+        'import/no-duplicates': 'error',
+        camelcase: 'warn',
```

Comment:
```suggestion
        camelcase: 'error',
```
Should change it to `error` once all the `warnings` are fixed.